hub/scripts.js

```diff
@@ -397,6 +397,39 @@
     });
   }
 
+  async function decoratePromotions() {
+    if (document.querySelector('main .promotion') instanceof HTMLElement) {
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector

The `querySelector` spec mentions:

> Returns: An Element object representing the first element in the document that matches the specified set of CSS selectors, or null is returned if there are no matches.

You can then safely simplifies this by 

```suggestion
    if (!document.querySelector('main .promotion')) {
```

Reply:
This is valid in other places. I realised you like this way of checking the `querySelector` method this way, it just looks much more complicated ;)

Reply:
I am aware of this and it makes sense. I think I've used it as a pattern after seeing some consumers overriding the `Element` object. This broke a lot of existing logic injected via JS.
The pattern does not affect performance and I'd like to keep it for now.